backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala

```diff
@@ -2096,5 +2096,37 @@ class GlutenClickHouseTPCHParquetSuite extends GlutenClickHouseTPCHAbstractSuite
     compareResultsAgainstVanillaSpark(select_sql, true, { _ => })
     spark.sql("drop table test_tbl_3135")
   }
+
+  test("GLUTEN-3134: Bug fix left join not match") {
```

Comment:
this test will never trigger https://github.com/oap-project/gluten/pull/3228/files#diff-2dd2fa3d120f6bd7b63ef5d48be5dfef3aead535054182d3cfba40d2f25a42f8R194 because right table will be broadcasted. consider using: withSQLConf(("spark.sql.autoBroadcastJoinThreshold", "1B"))

Reply:
done